src/cf-notifications/package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "cf-notification",
-  "version": "1.0.1",
-  "description": "Capital Framework notification",
-  "less": "src/cf-notification.less",
-  "style": "cf-notification.css",
+  "name": "cf-notifications",
+  "version": "1.0.2",
```

Comment:
Don't manually increment the version. When the bot opens the release PR it'll do it for you based on the changelog entry you added above.

Reply:
Fixed!